getting-started/getting-started.md

```diff
@@ -1,27 +1,1209 @@
+
+* [Eloquent Javascript](http://eloquentjavascript.net/ "Eloquent Javascript")
+* [Mozilla's Javascript page](https://developer.mozilla.org/en-US/docs/Web/JavaScript "Mozilla's Javascript page")
+* [W3C's Javascript tutorial](http://www.w3schools.com/js/default.asp "World Wide Web Consortium's Javascript tutorial")
+
+# Creating a New Sails Project
+
+The best way to use this guide is to follow each step as it happens, no code or step needed to make this example application has been left out, so you can literally follow along step by step.
+
+By following along with this guide, you'll create a Sails project called `blog`, a (very) simple weblog. Before you can start building the application, you need to make sure that you have Sails itself installed.
+
+*The examples below use $ to represent your terminal prompt in a UNIX-like OS, though it may have been customized to appear differently. If you are using Windows, your prompt will look something like `c:\source_code`>*
+
+
+## Installing Sails
 To install the latest stable release with the command-line tool:
 
-	sudo npm -g install sails
+    sudo npm -g install sails
 
 On Windows, you don't need sudo:
 
-	npm -g install sails
+    npm -g install sails
+
+# Creating the Blog Application
+
+Sails comes with a number of scripts called [generators](http://sailsjs.org/#/documentation/reference/cli/sailsgenerate.html "SailsJs generators") that are designed to make your development life easier by creating everything that's necessary to start working on a particular task. One of these is the [`new`](http://sailsjs.org/#/documentation/reference/cli/sailsnew.html "'sails new' generator") application generator, which will provide you with the foundation of a fresh Sails application so that you don't have to write it yourself.
+
+To use this generator, open a terminal, navigate to a directory where you have rights to create files, and type:
+
+    sails new --template jade blog
+
+This will create a Sails application called Blog in a blog directory and install the npm dependencies that are already mentioned in `package.json` using `npm install`.
+
+You can see all of the command line options that the Sails application builder accepts by running `sails new -h`.
+
+After you create the blog application, switch to its folder:
+
+    cd blog
+
+The `blog` directory has a number of auto-generated files and folders that make up the structure of a Sails application. The anatomy thereof is described [here](http://sailsjs.org/#/documentation/anatomy/myApp "Anatomy of a SailJs app").
+
+# Hello, Sails!
+
+To begin with, let's get some text up on screen quickly. To do this, you need to get your Sails application server running.
+
+## Starting up the Web Server
+
+You actually have a functional Sails application already. To see it, you need to start a web server on your development machine. You can do this by running the following in the blog directory:
+
+    sails lift
```

Comment:
Thanks for checking it out.

Updated with commit LoveIsGrief/sails-docs@f6583f923dac5d657252cd97fc05a0bef63e7cde